pkg/cloudprovider/vsphere/config_test.go

```diff
@@ -0,0 +1,48 @@
+package vsphere
```

Comment:
Can you add the license header here like the other files (dated 2019)?

Reply:
Done.